src/cli.rs

```diff
@@ -56,6 +56,14 @@ pub(crate) fn build_cli() -> Command<'static> {
                 .possible_values(&["text", "json", "otlp"])
                 .help("Log output format"),
         )
+        .arg(
+            Arg::new("log-no-color")
+                .long("log-no-color")
+                .takes_value(false)
+                .env("NO_COLOR")
```

Comment:
Should not be `KUBEWARDEN_LOG_NO_COLOR`?

Reply:
I changed it to be `NO_COLOR` as @viccuad suggested. There's seems to be a consensus about different cli tools to have this specific flag. See here: https://no-color.org/

Reply:
Right. Sorry. I missed the @viccuad comment.